packages/shared/declaration/index.ts

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/no-unassigned-import */
+import './react-app';
+import './dom';
+/* eslint-enable import/no-unassigned-import */
```

Comment:
We can actually import all the global scss files from this shared package, right? Like `normalized.scss`

Reply:
discussed offline, do it later